check-lint

```diff
@@ -74,6 +77,12 @@ if [ -f "$CONFIG_FILE" ]; then
                 echo "** Running $TOOL_NAME on directory '$DIRECTORY_PATH' with config from '$CONFIG_PATH'..."
                 $TOOL_CMD --rcfile="$BASE_DIR/../$CONFIG_PATH" $FILES
 
+                # Capture the exit code
+                exit_code=$?
+                if [ $exit_code -ne 0 ]; then
```

Comment:
What about moving this to line 78 like this:
```
if [  ! $TOOL_CMD --rcfile="$BASE_DIR/../$CONFIG_PATH" $FILES  ]; then
       overall_exit_status=1  # Set to 1 if there were any issues
fi
```

Reply:
Hi Jan,

Thanks for your comments, I agree, this would certainly simplify it.
Sometimes I over complicate when there are certainly easier solutions :)

I've made the changes as requested and tested them.